raiden/raiden_service.py

```diff
@@ -599,26 +598,24 @@ def _initialize_wal(self) -> None:
         state_change_qty = self.wal.storage.count_state_changes()
         self.snapshot_group = state_change_qty // SNAPSHOT_STATE_CHANGES_COUNT
 
-    def _log_sync_progress(self, to_block: BlockNumber) -> None:
+    def _log_sync_progress(self, current_block: BlockNumber, target_block: BlockNumber) -> None:
         """Print a message if there are many blocks to be fetched, or if the
         time in-between polls is high.
         """
         now = datetime.now()
-        blocks_to_sync = to_block - self.last_log_block
+        blocks_to_sync = target_block - current_block
         elapsed = (now - self.last_log_time).total_seconds()
 
         if blocks_to_sync > 100 or elapsed > 15.0:
             log.info(
                 "Synchronizing blockchain events",
                 blocks_to_sync=blocks_to_sync,
                 blocks_per_second=blocks_to_sync / elapsed,
-                to_block=to_block,
+                to_block=target_block,
                 elapsed=elapsed,
             )
             self.last_log_time = now
 
-        self.last_log_block = to_block
```

Comment:
After the first call the value was updated here, this too was not correct, since it updated the value to the *target* block number, which meant the progress was only showed once.